src/Dropdown/Dropdown.js

```diff
@@ -52,17 +52,26 @@ export default function Dropdown({
 }) {
   const popperRef = useRef();
   const menuRef = useRef();
+
   const [isOpen, setOpen] = useState(false);
+  const [isBlurable, setBlurable] = useState(false);
```

Comment:
Could this be assigned as a ref? State should generally be used in render, otherwise it can be a ref. We also benefit from the safety of the update processing synchronously, rather than waiting for next render

Reply:
useState has been replaced. I'm not familiar with using refs in that way, could you please elaborate?